### ISSUE TITLE
Support for static methods.

### ISSUE THREAD

**Issue by cmcculloh-kr** [egm0121/jest-test-gen#3]
For
```
class Variance {
    static getVector( style ) { ... }
}
```

Currently the following is created:

```
describe('Variance', () => {
	let instance;

	beforeEach(() => {
		instance = new Variance();
	});

	it('instance should be an instanceof Variance', () => {
		expect(instance instanceof Variance).toBeTruthy();
	});

	it('should have a method getVector()', () => {
		// instance.getVector(style);
		expect(false).toBeTruthy();
	});
});
```

But the following should be created instead:

```
describe('Variance', () => {
	let instance;

	beforeEach(() => {
		instance = new Variance();
	});

	it('instance should be an instanceof Variance', () => {
		expect(instance instanceof Variance).toBeTruthy();
	});

	it('should have a static method getVector()', () => {
		// Variance.getVector(style);
		expect(false).toBeTruthy();
	});
});
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static

**Comment by egm0121** [comment]
@cormacmccarthy  Agree, it should recognize it as a static method will fix in the next release

**Comment by egm0121** [comment]
@cormacmccarthy  this is now included in `jest-test-gen@1.1.1` :)
